src/matchers/toHaveBeenCalledOnce/index.js

```diff
@@ -0,0 +1,27 @@
+import { matcherHint, printReceived } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = mockCalls => () =>
+  matcherHint('.not.toHaveBeenCalledOnce') +
```

Comment:
As there is no expected value for this matcher you can change this to be:

```js
matcherHint('.not.toHaveBeenCalledOnce', 'received', '') +
```

Then the error will message will change from:

```
expect(received).not.toHaveBeenCalledOnce(expected)
```

to:

```
expect(received).not.toHaveBeenCalledOnce()
```

Reply:
didn't know about this one, thanks 💯